benchmark/hetero/at/fusion.vector.cpp.erb

```diff
@@ -2,24 +2,22 @@
 // Distributed under the Boost Software License, Version 1.0.
 // (See accompanying file LICENSE.md or copy at http://boost.org/LICENSE_1_0.txt)
 
-<% if n > 10 %>
-    #define FUSION_MAX_VECTOR_SIZE <%= ((n + 9) / 10) * 10 %>
```

Comment:
It was [never required](http://www.boost.org/doc/libs/1_62_0/libs/fusion/doc/html/fusion/container/vector.html):

> For C++11 compilers, the variadic function interface has no upper bound.